docs-src/docs/dataset-refinement/prediction-format.md

```diff
@@ -0,0 +1,188 @@
+---
+title: The prediction format
+---
+
+Predictions use the OpenLabel format/schema. This is the same format as the one used
+for [uploading pre-annotations](../kognic-io/pre_annotations/#openlabel-support). General information about the
+OpenLabel format can be found in [here](../openlabel/openlabel-format).
+
+## Supported prediction features
+
+:::note
+Only one type of feature per prediction is supported.
```

Comment:
I didn't understand what a feature was when I read this, maybe change it to "Only one type of feature, for example cuboid in 3D, per prediction is supported" or something similar

Reply:
Definitely. Agree!